src/util.c

```diff
@@ -690,6 +690,63 @@ create_cachedirtag(const char *dir)
 	return 1;
 }
 
+#ifdef _WIN32
+// Replace forward slashes with backslashes
+// Fixes issues when path got backslashes AND forward slashes
+char* 
+slash_to_backslash(char* str)
```

Comment:
Wouldn't this be way simpler with std::string and replace? I know it's char* all around, but it should be std::string anyway.